x-pack/plugins/security/server/plugin.ts

```diff
@@ -74,23 +77,29 @@ export interface SecurityPluginSetup {
   /**
    * @deprecated Use `authz` methods from the `SecurityServiceStart` contract instead.
    */
-  authz: Pick<
```

Comment:
__question:__ I definitely agree that it's much better to use dedicated interface here, but just out of curiosity - does `Pick` not play well with the API docs?

Reply:
It doesn't, it also doesn't work well in an IDE. From best practices:

<img width="905" alt="Screen Shot 2021-06-09 at 8 38 12 AM" src="https://user-images.githubusercontent.com/16563603/121355630-1127c700-c8fe-11eb-9895-afee910ec56a.png">

Reply:
Ah, missed that somehow, good to know, thanks!